src/components/ReanimatedSwipeable.tsx

```diff
@@ -334,37 +364,20 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
           },
           (isFinished) => {
             if (isFinished) {
-              if (toValue > 0 && props.onSwipeableOpen) {
-                runOnJS(props.onSwipeableOpen)(
-                  'left',
-                  swipeableMethods.current
-                );
-              } else if (toValue < 0 && props.onSwipeableOpen) {
-                runOnJS(props.onSwipeableOpen)(
-                  'right',
-                  swipeableMethods.current
-                );
-              } else if (props.onSwipeableClose) {
-                const closingDirection = fromValue > 0 ? 'left' : 'right';
-                runOnJS(props.onSwipeableClose)(
-                  closingDirection,
-                  swipeableMethods.current
-                );
-              }
+              dispatchEndEvents(fromValue, toValue);
```

Comment:
```suggestion
              runOnJS(dispatchEndEvents)(fromValue, toValue);
```

Since `dispatchEndEvents` isn't a worklet (and we probably don't want it to be), you need `runOnJS` here.